[2024-12-10T18:00:40.144-0600] {processor.py:186} INFO - Started process (PID=5031) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:00:40.145-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:00:40.146-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:00:40.145-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:00:40.148-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:00:40.147-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:00:40.148-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:00:40.148-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:00:40.148-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:00:40.155-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-12-10T18:03:01.239-0600] {processor.py:186} INFO - Started process (PID=5243) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:03:01.240-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:03:01.241-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:03:01.241-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:03:01.243-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:03:01.243-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:03:01.244-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:03:01.244-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:03:01.244-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:03:01.251-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-12-10T18:03:56.271-0600] {processor.py:186} INFO - Started process (PID=5318) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:03:56.275-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:03:56.277-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:03:56.276-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:03:56.279-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:03:56.279-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:03:56.280-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:03:56.279-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:03:56.280-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:03:56.296-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2024-12-10T18:04:54.855-0600] {processor.py:186} INFO - Started process (PID=5391) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:04:54.856-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:04:54.858-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:04:54.858-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:04:54.860-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:04:54.860-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:04:54.860-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:04:54.860-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:04:54.861-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:04:54.868-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-12-10T18:05:52.238-0600] {processor.py:186} INFO - Started process (PID=5555) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:05:52.240-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:05:52.242-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:05:52.242-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:05:52.244-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:05:52.244-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:05:52.244-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:05:52.244-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:05:52.244-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:05:52.252-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:06:49.011-0600] {processor.py:186} INFO - Started process (PID=5617) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:06:49.013-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:06:49.014-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:06:49.014-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:06:49.017-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:06:49.016-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:06:49.017-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:06:49.017-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:06:49.017-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:06:49.026-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-12-10T18:07:49.933-0600] {processor.py:186} INFO - Started process (PID=5719) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:07:49.935-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:07:49.936-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:07:49.936-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:07:49.939-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:07:49.938-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:07:49.939-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:07:49.939-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:07:49.939-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:07:49.946-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-12-10T18:08:52.147-0600] {processor.py:186} INFO - Started process (PID=5841) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:08:52.149-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:08:52.150-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:08:52.150-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:08:52.153-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:08:52.152-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:08:52.153-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:08:52.153-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:08:52.153-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:08:52.160-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-12-10T18:13:26.714-0600] {processor.py:186} INFO - Started process (PID=6008) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:13:26.717-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:13:26.719-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:13:26.719-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:13:26.722-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:13:26.722-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:13:26.723-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:13:26.722-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:13:26.723-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:13:26.733-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-12-10T18:14:29.861-0600] {processor.py:186} INFO - Started process (PID=6083) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:14:29.863-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:14:29.866-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:14:29.865-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:14:29.868-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:14:29.868-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:14:29.876-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:14:29.875-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:14:29.876-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:14:29.887-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2024-12-10T18:15:26.649-0600] {processor.py:186} INFO - Started process (PID=6147) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:15:26.652-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:15:26.653-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:15:26.653-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:15:26.656-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:15:26.656-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:15:26.657-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:15:26.657-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:15:26.657-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:15:26.664-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T18:17:24.977-0600] {processor.py:186} INFO - Started process (PID=6388) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:17:24.978-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:17:24.980-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:17:24.980-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:17:24.982-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:17:24.982-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:17:24.982-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:17:24.982-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:17:24.982-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:17:24.990-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-12-10T18:18:33.886-0600] {processor.py:186} INFO - Started process (PID=6619) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:18:33.888-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:18:33.889-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:18:33.889-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:18:33.892-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:18:33.891-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:18:33.892-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:18:33.892-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:18:33.892-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:18:33.899-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-12-10T18:19:39.682-0600] {processor.py:186} INFO - Started process (PID=6690) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:19:39.685-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:19:39.686-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:19:39.686-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:19:39.688-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:19:39.688-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:19:39.689-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:19:39.689-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:19:39.689-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:19:39.696-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:20:35.507-0600] {processor.py:186} INFO - Started process (PID=6751) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:20:35.510-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:20:35.512-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:20:35.512-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:20:35.515-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:20:35.514-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:20:35.515-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:20:35.515-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:20:35.515-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:20:35.522-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T18:21:30.143-0600] {processor.py:186} INFO - Started process (PID=6811) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:21:30.146-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:21:30.147-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:21:30.147-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:21:30.149-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:21:30.149-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:21:30.150-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:21:30.150-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:21:30.150-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:21:30.157-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:22:23.753-0600] {processor.py:186} INFO - Started process (PID=6871) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:22:23.756-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:22:23.758-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:22:23.757-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:22:23.760-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:22:23.760-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:22:23.760-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:22:23.760-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:22:23.760-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:22:23.767-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:23:16.333-0600] {processor.py:186} INFO - Started process (PID=6929) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:23:16.336-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:23:16.338-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:23:16.337-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:23:16.340-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:23:16.340-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:23:16.340-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:23:16.340-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:23:16.340-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:23:16.348-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T18:24:10.484-0600] {processor.py:186} INFO - Started process (PID=6985) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:24:10.491-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:24:10.493-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:24:10.492-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:24:10.495-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:24:10.495-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:24:10.495-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:24:10.495-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:24:10.495-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:24:10.503-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-12-10T18:25:04.039-0600] {processor.py:186} INFO - Started process (PID=7048) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:25:04.040-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:25:04.042-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:25:04.042-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:25:04.045-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:25:04.044-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:25:04.045-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:25:04.045-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:25:04.045-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:25:04.054-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2024-12-10T18:26:19.985-0600] {processor.py:186} INFO - Started process (PID=7132) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:26:19.986-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:26:19.988-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:26:19.988-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:26:19.991-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:26:19.990-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:26:19.991-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:26:19.991-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:26:19.991-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:26:20.000-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T18:27:40.257-0600] {processor.py:186} INFO - Started process (PID=7218) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:27:40.258-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:27:40.260-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:27:40.260-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:27:40.263-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:27:40.263-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:27:40.264-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:27:40.263-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:27:40.264-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:27:40.272-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T18:28:34.150-0600] {processor.py:186} INFO - Started process (PID=7276) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:28:34.151-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:28:34.153-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:28:34.153-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:28:34.155-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:28:34.155-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:28:34.156-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:28:34.156-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:28:34.156-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:28:34.163-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-12-10T18:29:31.550-0600] {processor.py:186} INFO - Started process (PID=7335) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:29:31.552-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:29:31.554-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:29:31.554-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:29:31.556-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:29:31.556-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:29:31.556-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:29:31.556-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:29:31.557-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:29:31.564-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-12-10T18:30:27.101-0600] {processor.py:186} INFO - Started process (PID=7395) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:30:27.103-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:30:27.105-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:30:27.105-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:30:27.108-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:30:27.107-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:30:27.108-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:30:27.108-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:30:27.108-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:30:27.115-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:31:20.742-0600] {processor.py:186} INFO - Started process (PID=7453) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:31:20.748-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:31:20.750-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:31:20.749-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:31:20.752-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:31:20.752-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:31:20.752-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:31:20.752-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:31:20.752-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:31:20.759-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-12-10T18:32:13.817-0600] {processor.py:186} INFO - Started process (PID=7511) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:32:13.818-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:32:13.820-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:32:13.820-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:32:13.822-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:32:13.822-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:32:13.822-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:32:13.822-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:32:13.823-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:32:13.830-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-12-10T18:33:07.747-0600] {processor.py:186} INFO - Started process (PID=7569) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:33:07.749-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:33:07.751-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:33:07.751-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:33:07.753-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:33:07.753-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:33:07.753-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:33:07.753-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:33:07.754-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:33:07.761-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:34:23.464-0600] {processor.py:186} INFO - Started process (PID=7652) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:34:23.467-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:34:23.468-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:34:23.468-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:34:23.470-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:34:23.470-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:34:23.470-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:34:23.470-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:34:23.470-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:34:23.478-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:35:40.843-0600] {processor.py:186} INFO - Started process (PID=7740) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:35:40.845-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:35:40.847-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:35:40.847-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:35:40.850-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:35:40.849-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:35:40.850-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:35:40.850-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:35:40.850-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:35:40.857-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:36:34.026-0600] {processor.py:186} INFO - Started process (PID=7799) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:36:34.028-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:36:34.030-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:36:34.029-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:36:34.032-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:36:34.032-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:36:34.032-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:36:34.032-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:36:34.032-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:36:34.040-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:37:27.945-0600] {processor.py:186} INFO - Started process (PID=7856) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:37:27.948-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:37:27.950-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:37:27.950-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:37:27.952-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:37:27.952-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:37:27.952-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:37:27.952-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:37:27.953-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:37:27.960-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:38:22.172-0600] {processor.py:186} INFO - Started process (PID=7915) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:38:22.175-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:38:22.180-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:38:22.179-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:38:22.182-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:38:22.182-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:38:22.182-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:38:22.182-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:38:22.183-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:38:22.190-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2024-12-10T18:39:14.190-0600] {processor.py:186} INFO - Started process (PID=7972) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:39:14.192-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:39:14.194-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:39:14.194-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:39:14.196-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:39:14.196-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:39:14.196-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:39:14.196-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:39:14.196-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:39:14.204-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:40:09.562-0600] {processor.py:186} INFO - Started process (PID=8061) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:40:09.564-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:40:09.566-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:40:09.566-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:40:09.568-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:40:09.568-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:40:09.568-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:40:09.568-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:40:09.568-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:40:09.576-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:41:24.820-0600] {processor.py:186} INFO - Started process (PID=8564) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:41:24.822-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:41:24.824-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:41:24.823-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:41:24.826-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:41:24.825-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:41:24.826-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:41:24.826-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:41:24.826-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:41:24.834-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:42:41.442-0600] {processor.py:186} INFO - Started process (PID=8649) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:42:41.445-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:42:41.446-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:42:41.446-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:42:41.449-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:42:41.448-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:42:41.449-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:42:41.449-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:42:41.449-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:42:41.456-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:43:37.367-0600] {processor.py:186} INFO - Started process (PID=8708) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:43:37.370-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:43:37.372-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:43:37.372-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:43:37.374-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:43:37.374-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:43:37.374-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:43:37.374-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:43:37.375-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:43:37.382-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:44:32.987-0600] {processor.py:186} INFO - Started process (PID=8773) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:44:32.989-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:44:32.991-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:44:32.991-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:44:32.993-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:44:32.993-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:44:32.994-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:44:32.994-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:44:32.994-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:44:33.001-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T18:45:51.348-0600] {processor.py:186} INFO - Started process (PID=8864) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:45:51.350-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:45:51.352-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:45:51.352-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:45:51.355-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:45:51.354-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:45:51.355-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:45:51.355-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:45:51.355-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:45:51.365-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2024-12-10T18:46:45.568-0600] {processor.py:186} INFO - Started process (PID=8924) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:46:45.571-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:46:45.572-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:46:45.572-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:46:45.575-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:46:45.574-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:46:45.575-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:46:45.575-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:46:45.575-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:46:45.584-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-12-10T18:48:02.058-0600] {processor.py:186} INFO - Started process (PID=9008) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:48:02.061-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:48:02.063-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:48:02.062-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:48:02.065-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:48:02.065-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:48:02.065-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:48:02.065-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:48:02.066-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:48:02.073-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T18:48:54.658-0600] {processor.py:186} INFO - Started process (PID=9065) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:48:54.661-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:48:54.663-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:48:54.663-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:48:54.665-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:48:54.665-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:48:54.666-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:48:54.665-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:48:54.666-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:48:54.673-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:49:49.733-0600] {processor.py:186} INFO - Started process (PID=9123) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:49:49.734-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:49:49.736-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:49:49.736-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:49:49.738-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:49:49.738-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:49:49.739-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:49:49.739-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:49:49.739-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:49:49.747-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:50:41.416-0600] {processor.py:186} INFO - Started process (PID=9182) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:50:41.418-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:50:41.420-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:50:41.420-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:50:41.422-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:50:41.422-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:50:41.423-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:50:41.422-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:50:41.423-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:50:41.431-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T18:51:36.190-0600] {processor.py:186} INFO - Started process (PID=9241) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:51:36.193-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:51:36.194-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:51:36.194-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:51:36.197-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:51:36.196-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:51:36.197-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:51:36.197-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:51:36.197-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:51:36.205-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:52:34.064-0600] {processor.py:186} INFO - Started process (PID=9300) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:52:34.067-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:52:34.068-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:52:34.068-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:52:34.071-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:52:34.070-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:52:34.071-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:52:34.071-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:52:34.071-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:52:34.078-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:53:30.070-0600] {processor.py:186} INFO - Started process (PID=9360) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:53:30.072-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:53:30.074-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:53:30.074-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:53:30.076-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:53:30.076-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:53:30.077-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:53:30.077-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:53:30.077-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:53:30.084-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:54:47.267-0600] {processor.py:186} INFO - Started process (PID=9445) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:54:47.269-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:54:47.271-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:54:47.271-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:54:47.273-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:54:47.273-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:54:47.273-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:54:47.273-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:54:47.274-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:54:47.281-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T18:56:02.053-0600] {processor.py:186} INFO - Started process (PID=9531) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:56:02.055-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:56:02.057-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:56:02.056-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:56:02.059-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:56:02.058-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:56:02.059-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:56:02.059-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:56:02.059-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:56:02.067-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-12-10T18:56:57.499-0600] {processor.py:186} INFO - Started process (PID=9591) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:56:57.502-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:56:57.503-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:56:57.503-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:56:57.507-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:56:57.506-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:56:57.507-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:56:57.507-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:56:57.507-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:56:57.515-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T18:57:53.910-0600] {processor.py:186} INFO - Started process (PID=9650) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:57:53.913-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:57:53.914-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:57:53.914-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:57:53.917-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:57:53.917-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:57:53.917-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:57:53.917-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:57:53.918-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:57:53.925-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T18:59:11.771-0600] {processor.py:186} INFO - Started process (PID=9734) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:59:11.773-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T18:59:11.775-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:59:11.774-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:59:11.777-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:59:11.777-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T18:59:11.778-0600] {logging_mixin.py:190} INFO - [2024-12-10T18:59:11.778-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T18:59:11.778-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T18:59:11.786-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T19:00:09.746-0600] {processor.py:186} INFO - Started process (PID=9793) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:00:09.748-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:00:09.750-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:00:09.749-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:00:09.752-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:00:09.752-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:00:09.752-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:00:09.752-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:00:09.752-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:00:09.760-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T19:01:08.296-0600] {processor.py:186} INFO - Started process (PID=9855) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:01:08.299-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:01:08.300-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:01:08.300-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:01:08.303-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:01:08.302-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:01:08.303-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:01:08.303-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:01:08.303-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:01:08.311-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T19:02:05.105-0600] {processor.py:186} INFO - Started process (PID=9917) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:02:05.107-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:02:05.109-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:02:05.109-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:02:05.111-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:02:05.111-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:02:05.112-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:02:05.111-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:02:05.112-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:02:05.119-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T19:02:59.146-0600] {processor.py:186} INFO - Started process (PID=9976) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:02:59.149-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:02:59.150-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:02:59.150-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:02:59.152-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:02:59.152-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:02:59.153-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:02:59.153-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:02:59.153-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:02:59.160-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T19:03:54.655-0600] {processor.py:186} INFO - Started process (PID=10034) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:03:54.657-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:03:54.659-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:03:54.658-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:03:54.661-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:03:54.660-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:03:54.661-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:03:54.661-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:03:54.661-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:03:54.669-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T19:04:51.528-0600] {processor.py:186} INFO - Started process (PID=10094) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:04:51.531-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:04:51.533-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:04:51.532-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:04:51.535-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:04:51.535-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:04:51.536-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:04:51.536-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:04:51.536-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:04:51.544-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-12-10T19:06:10.260-0600] {processor.py:186} INFO - Started process (PID=10183) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:06:10.263-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:06:10.265-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:06:10.265-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:06:10.267-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:06:10.267-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:06:10.268-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:06:10.267-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:06:10.268-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:06:10.275-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T19:07:23.799-0600] {processor.py:186} INFO - Started process (PID=10267) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:07:23.801-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:07:23.802-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:07:23.802-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:07:23.805-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:07:23.804-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:07:23.805-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:07:23.805-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:07:23.805-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:07:23.813-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T19:08:40.926-0600] {processor.py:186} INFO - Started process (PID=10351) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:08:40.928-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:08:40.930-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:08:40.930-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:08:40.933-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:08:40.932-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:08:40.933-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:08:40.933-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:08:40.933-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:08:40.940-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T19:09:36.637-0600] {processor.py:186} INFO - Started process (PID=10411) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:09:36.639-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:09:36.641-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:09:36.641-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:09:36.644-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:09:36.643-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:09:36.644-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:09:36.644-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:09:36.644-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:09:36.652-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T19:10:31.753-0600] {processor.py:186} INFO - Started process (PID=10469) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:10:31.755-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:10:31.757-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:10:31.757-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:10:31.759-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:10:31.759-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:10:31.760-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:10:31.759-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:10:31.760-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:10:31.767-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T19:11:41.420-0600] {processor.py:186} INFO - Started process (PID=10550) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:11:41.421-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:11:41.424-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:11:41.423-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:11:41.426-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:11:41.426-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:11:41.427-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:11:41.427-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:11:41.427-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:11:41.436-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-12-10T19:12:56.168-0600] {processor.py:186} INFO - Started process (PID=10633) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:12:56.170-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:12:56.172-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:12:56.172-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:12:56.174-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:12:56.174-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:12:56.175-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:12:56.174-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:12:56.175-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:12:56.182-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T19:13:48.645-0600] {processor.py:186} INFO - Started process (PID=10690) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:13:48.648-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:13:48.650-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:13:48.649-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:13:48.652-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:13:48.651-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:13:48.652-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:13:48.652-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:13:48.652-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:13:48.660-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T19:15:06.504-0600] {processor.py:186} INFO - Started process (PID=10777) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:15:06.506-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:15:06.508-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:15:06.508-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:15:06.511-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:15:06.510-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:15:06.511-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:15:06.511-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:15:06.511-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:15:06.521-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-12-10T19:16:00.110-0600] {processor.py:186} INFO - Started process (PID=10834) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:16:00.115-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:16:00.118-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:16:00.118-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:16:00.125-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:16:00.123-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:16:00.126-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:16:00.126-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:16:00.126-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:16:00.139-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2024-12-10T19:16:52.163-0600] {processor.py:186} INFO - Started process (PID=10892) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:16:52.165-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:16:52.166-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:16:52.166-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:16:52.169-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:16:52.168-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:16:52.169-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:16:52.169-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:16:52.170-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:16:52.178-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T19:17:45.920-0600] {processor.py:186} INFO - Started process (PID=10953) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:17:45.923-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:17:45.924-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:17:45.924-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:17:45.927-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:17:45.926-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:17:45.927-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:17:45.927-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:17:45.927-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:17:45.936-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2024-12-10T19:18:39.075-0600] {processor.py:186} INFO - Started process (PID=11012) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:18:39.076-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:18:39.078-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:18:39.078-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:18:39.080-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:18:39.080-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:18:39.081-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:18:39.080-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:18:39.081-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:18:39.088-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-12-10T19:19:56.638-0600] {processor.py:186} INFO - Started process (PID=11095) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:19:56.640-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:19:56.642-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:19:56.642-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:19:56.644-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:19:56.644-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:19:56.645-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:19:56.645-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:19:56.645-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:19:56.652-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T19:20:51.624-0600] {processor.py:186} INFO - Started process (PID=11154) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:20:51.626-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:20:51.628-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:20:51.628-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:20:51.631-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:20:51.630-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:20:51.631-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:20:51.631-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:20:51.631-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:20:51.638-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T19:22:07.490-0600] {processor.py:186} INFO - Started process (PID=11239) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:22:07.493-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:22:07.495-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:22:07.495-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:22:07.498-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:22:07.497-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:22:07.498-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:22:07.498-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:22:07.498-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:22:07.506-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T19:23:00.055-0600] {processor.py:186} INFO - Started process (PID=11297) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:23:00.056-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:23:00.058-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:23:00.058-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:23:00.060-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:23:00.060-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:23:00.061-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:23:00.060-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:23:00.061-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:23:00.068-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-12-10T19:24:17.243-0600] {processor.py:186} INFO - Started process (PID=11381) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:24:17.245-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:24:17.246-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:24:17.246-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:24:17.249-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:24:17.248-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:24:17.249-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:24:17.249-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:24:17.249-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:24:17.257-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T19:25:38.385-0600] {processor.py:186} INFO - Started process (PID=11470) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:25:38.387-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:25:38.389-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:25:38.389-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:25:38.391-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:25:38.391-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:25:38.392-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:25:38.391-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:25:38.392-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:25:38.399-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T19:26:33.391-0600] {processor.py:186} INFO - Started process (PID=11529) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:26:33.393-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:26:33.395-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:26:33.394-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:26:33.397-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:26:33.397-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:26:33.397-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:26:33.397-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:26:33.397-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:26:33.405-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T19:27:48.210-0600] {processor.py:186} INFO - Started process (PID=11614) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:27:48.212-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:27:48.214-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:27:48.213-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:27:48.217-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:27:48.216-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:27:48.217-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:27:48.217-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:27:48.217-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:27:48.224-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T19:28:41.528-0600] {processor.py:186} INFO - Started process (PID=11672) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:28:41.530-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:28:41.532-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:28:41.532-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:28:41.535-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:28:41.534-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:28:41.535-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:28:41.535-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:28:41.535-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:28:41.542-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T19:29:38.509-0600] {processor.py:186} INFO - Started process (PID=11731) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:29:38.512-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:29:38.514-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:29:38.514-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:29:38.516-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:29:38.516-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:29:38.516-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:29:38.516-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:29:38.517-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:29:38.524-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-10T19:30:53.915-0600] {processor.py:186} INFO - Started process (PID=11814) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:30:53.917-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:30:53.918-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:30:53.918-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:30:53.920-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:30:53.920-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:30:53.921-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:30:53.920-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:30:53.921-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:30:53.929-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-12-10T19:31:50.137-0600] {processor.py:186} INFO - Started process (PID=11875) to work on /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:31:50.139-0600] {processor.py:914} INFO - Processing file /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-10T19:31:50.141-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:31:50.140-0600] {dagbag.py:588} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:31:50.143-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:31:50.143-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-10T19:31:50.144-0600] {logging_mixin.py:190} INFO - [2024-12-10T19:31:50.144-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-10T19:31:50.144-0600] {processor.py:927} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-10T19:31:50.151-0600] {processor.py:208} INFO - Processing /root/airflow_env/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
